### DISCUSSION TITLE
Background color footer where to set?

### DISCUSSION THREAD

**Discussion by Gruhler** [GetPublii/Publii#1581] [Help]
Hi,
Background color footer where to set?
Theme Tattoo ( what I try it is always black)
Paul

**Comment by candidexmedia** [comment]
If the theme doesn't have a specific [Theme Setting](https://getpublii.com/docs/theme-settings.html) for the footer colour, use [Tools & Plugins > Custom CSS](https://getpublii.com/docs/tools.html#customcss) to override the colour:

```
.footer {
background: #ffffff /*set the colour that you want using hex codes, rgb, etc.*/
}
```

Change the `#ffffff` [color value](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value) with the HEX, RGB, HSL, hwb, lab, lch named colour, CSS colour variable, etc. of your choice. Save your changes, and Preview/Publish your site.

Here's where it appears in the original theme:

![ Publii change footer colour for Tattoo theme](https://github.com/user-attachments/assets/59398236-6480-4e23-b14a-5674b2beab5d)

**Reply by candidexmedia** [reply]
If you're using Chrome or a Chromium-based browser, here's how to copy those changes easily using the Inspect Tool: https://i.imgur.com/YDAI016.mp4

![Copy CSS changes easily Chrome](https://github.com/user-attachments/assets/d173f0b1-a8f7-4280-ae12-b72e2cadc1ad)


For Firefox: https://devtoolstips.org/tips/en/find-css-changes/

![image](https://github.com/user-attachments/assets/c53d3fab-fce8-496c-8c7c-2f713b3b7f3b)

**Reply by Gruhler** [reply]
Wow --- thank you
it works wonderfully
